torch/csrc/jit/script/tree_views.h

```diff
@@ -893,17 +893,18 @@ struct SliceExpr : public Expr {
   Maybe<Expr> step() const {
     return Maybe<Expr>(subtree(2));
   }
-  Expr startOr(int alternative) const {
+  Expr startOr(int64_t alternative) const {
     const auto startOption = start();
     return startOption.present() ? startOption.get() : createInt(alternative);
   }
-  Expr endOr(int alternative) const {
+  Expr endOr(int64_t alternative) const {
     const auto endOption = end();
     return endOption.present() ? endOption.get() : createInt(alternative);
   }
-  Expr stepOr(int alternative) const {
+  Expr stepOr(int64_t alternative) const {
     const auto stepOption = step();
-    return stepOption.present() ? stepOption.get() : createInt(alternative);
+    auto ret = stepOption.present() ? stepOption.get() : createInt(alternative);
```

Comment:
nit: What is the reason for this change? Just linting?

Reply:
Removed old debugging